site/src/pages/RoadmapPage/sidebar/ProgramRequirementsList.scss

```diff
@@ -80,8 +78,8 @@ $complete-color: green;
   display: flex;
   justify-content: center;
   align-items: center;
-  padding: 5px 2px;
-  background-color: globals.$primary-blue-1;
+  padding: 5px 10px;
```

Comment:
I assume this happened because of a merge conflict, but please revert this line

Reply:
"this line" refers to just the padding. for some reason GH did not highlight correctly